wdl/CramToBam.wdl

```diff
@@ -1,15 +1,20 @@
 version 1.0
 
 import "Structs.wdl"
+import "Utils.wdl" as util
 
 workflow CramToBam {
   input {
     File cram_file
     File reference_fasta
     File? reference_index
+    String? service_account_json  # Provide path to service account credentials JSON if required to access CRAM file
```

Comment:
Make a note that neither the json nor cram can be in requester pays buckets in this case